src/worker/core/draft/genPlayersWithoutSaving.ts

```diff
@@ -33,12 +33,11 @@ const genPlayersWithoutSaving = async (
 		forceScrubs = numRealPlayers > 0.5 * normalNumPlayers;
 	}
 
-	const baseAge =
-		bySport({
-			hockey: 18,
-			default: 19,
-		}) -
-		(draftYear - g.get("season"));
+	const draftAge = g.get("draftAge");
+	const baseAge = !isSport("football")
```

Comment:
Add a comment like:

    // The distribution of football rookies in the default FBGM was written to assume they were generated at 19 and then developed for 2 seasons. If `draftAge` existed when that code was written, it would not have made sense to include those 2 seasons. But doing something about that is much more difficult, so we want to keep the 2 seasons of extra development for FBGM currently.

Otherwise this will be very confusing to look at in the future!